### ISSUE TITLE
Has anyone gotten flash attention to work with lora adapters? (mistral 7b)

### ISSUE THREAD

**Issue by emilypyq** [Dao-AILab/flash-attention#636]
I'm trying to load various mistral 7b lora adapters (with [this ](https://github.com/Dao-AILab/flash-attention)) and was hoping to get faster inference with flash attention.  However, the inference speed is nearly identical to the adapter running without flash attention.  Has anyone else gotten this to work for a similar scenario?

**Comment by cassianlewis** [comment]
Yeah done this exactly like https://huggingface.co/docs/transformers/perf_infer_gpu_one#flash-attention-2
Getting virtually the same results
